### PR TITLE
use JavaScript classes

### PR DESCRIPTION
**Pull request by medaminedev66**
- Add 2 classes: `Book` and `Books`,
- add a method `addBook` to the Books class

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -29,13 +39,9 @@ if (localStorage.getItem('booksData') !== null) {
...
-  const newbook = new Book(titleInput, authorInput);
-
   if (localStorage.getItem('booksData') === null) {
     localStorage.setItem('booksData', JSON.stringify([]));
   }
-  const oldData = JSON.parse(localStorage.getItem('booksData'));
-  oldData.push(newbook);
-  localStorage.setItem('booksData', JSON.stringify(oldData));
+  const newbook = new Books();
+  newbook.addBook(titleInput, authorInput);
```
Comment (mikenath223) [inline index.js:46]: - [ ] OPTIONAL - Consider simply wrapping the functionality to delete a book into a function. It keeps the code functional and modular.

File: index.js [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
+    localStorage.setItem('booksData', JSON.stringify(booksData));
+  }
 }
 if (localStorage.getItem('booksData') !== null) {
   const booksData = JSON.parse(localStorage.getItem('booksData'));
-  booksData.forEach((book) => {
+  booksData.forEach((book, id) => {
     const bookContainer = document.createElement('div');
     const bookTitle = document.createElement('p');
     const bookAuthor = document.createElement('p');
```
Comment (henatan99) [inline index.js:29]: - [ ] OPTIONAL: You can add the DOM rendering of booksData in a render() class function in Books class.

**Review by henatan99 — APPROVED** [review]
Hi @emmyobonyo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 2 above)
Comment (medaminedev66) [reply index.js:29]: Thank you for the suggestion, it helped me a lot to organize the Books class and use fewer code lines.

**Pull request merged**